pkg/auto/release.go

```diff
@@ -0,0 +1,181 @@
+package auto
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/rancher/charts-build-scripts/pkg/filesystem"
+	"github.com/rancher/charts-build-scripts/pkg/git"
+	"github.com/rancher/charts-build-scripts/pkg/lifecycle"
+	"github.com/rancher/charts-build-scripts/pkg/path"
+	"gopkg.in/yaml.v2"
+)
+
+// Release holds necessary metadata to release a chart version
+type Release struct {
```

Comment:
Is this used outside of this package? If not, please unexport it.

Reply:
It is not yet used but will. 

Either way, I am converting it to a private struct.